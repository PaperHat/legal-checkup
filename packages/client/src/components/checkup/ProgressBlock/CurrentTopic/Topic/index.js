import styled from 'styled-components'

const Topic = styled.div`
  background-color: #ededed;
  color: #000000;
  font-weight: bold;
  height: 42px;
  line-height: 42px;
  margin: 0 1px;
  padding: 0 10px;
  text-align: center;
  width: 158px;
`

Topic.displayName = 'Topic'

export default Topic
