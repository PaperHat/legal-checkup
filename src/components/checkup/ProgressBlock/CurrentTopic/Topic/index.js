import styled from 'styled-components'

const Topic = styled.div`
  color: #b62645;

  font-family: 'Museo Sans Cyrl 900', sans-serif;
  font-weight: 900;
  font-size: 17px;

  text-align: center;
  text-transform: uppercase;
  white-space: nowrap;
  text-overflow: ellipsis;
  overflow: hidden;
`

Topic.displayName = 'Topic'

export default Topic
