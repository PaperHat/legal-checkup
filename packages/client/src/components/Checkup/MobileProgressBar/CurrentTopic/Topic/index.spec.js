import React from 'react'

import { shallow } from 'enzyme'

import Topic from './'

describe('components:styled:Topic', () => {
  it('renders the Topic and CSS properties properly', () => {
    expect(toJson(shallow(<Topic>This Topic</Topic>))).toMatchSnapshot()
  })
})
