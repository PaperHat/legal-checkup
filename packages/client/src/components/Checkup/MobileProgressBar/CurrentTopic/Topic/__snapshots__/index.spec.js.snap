// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:Topic renders the Topic and CSS properties properly 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  background-color: #ededed;
  color: #000000;
  font-weight: bold;
  height: 42px;
  line-height: 42px;
  margin: 0 1px;
  padding: 0 10px;
  text-align: center;
  width: 158px;
",
        ],
      },
      "displayName": "Topic",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  This Topic
</StyledComponent>
`;
